backend/src/main/java/com/carffeine/carffeine/service/chargerStation/dto/RandomKeySelector.java

```diff
@@ -0,0 +1,18 @@
+package com.carffeine.carffeine.service.chargerStation.dto;
+
+import org.springframework.beans.factory.annotation.Value;
+
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class RandomKeySelector {
+
+    private static final ThreadLocalRandom threadLocalRandom = ThreadLocalRandom.current();
+    @Value("${api.service_key}")
+    private static List<String> serviceKeys;
+
+    public static String generateRandomKey() {
+        int randomNumber = threadLocalRandom.nextInt(0, 7);
+        return serviceKeys.get(randomNumber);
+    }
```

Comment:
이 클래스를 빈으로 등록하면 좋을 것 같아요!
```suggestion
@Component
public class RandomKeySelector {

    private static final ThreadLocalRandom threadLocalRandom = ThreadLocalRandom.current();

    @Value("${api.service_key}")
    private List<String> serviceKeys;

    public String generateRandomKey() {
        int randomNumber = threadLocalRandom.nextInt(0, 7);
        return serviceKeys.get(randomNumber);
    }
```